frontend/src/app/game/graphs/graphs.component.ts

```diff
@@ -31,160 +31,183 @@ export class GraphsComponent implements AfterViewInit {
-      ),
+    this.immunized$ = this.immunizedChart$.pipe(
+      filter(states => states.length > 0),
+      map(states => states[states.length - 1]),
+      map(state => state[0].value + state[1].value),
     );
 
     this.activeTab = 0;
 
     this.templateData = [
       {
         label: 'Nově nakažení',
-        icon: 'virus' as SvgIconName,
-        headerData$: this.infectedToday$.pipe(map(gs => gs.value)),
-        data$: this.infectedToday$ as unknown as Observable<ChartValue>,
+        icon: 'virus',
+        headerData$: this.infectedToday$.pipe(map(gs => last(gs)?.value)),
+        data$: this.infectedToday$,
         customOptions: null,
         pipe: [false, false],
       },
       {
         label: 'Nově zemřelí',
-        icon: 'skull' as SvgIconName,
-        headerData$: this.deathToday$.pipe(map(gs => gs.value)),
-        data$: this.deathToday$ as unknown as Observable<ChartValue>,
+        icon: 'skull',
+        headerData$: this.deathToday$.pipe(map(gs => last(gs)?.value)),
+        data$: this.deathToday$,
         customOptions: null,
         pipe: [false, false],
       },
       {
         label: 'Celkové náklady',
-        icon: 'money' as SvgIconName,
-        headerData$: this.costTotal$.pipe(map(gs => gs.value)),
-        data$: this.costTotal$ as unknown as Observable<ChartValue>,
+        icon: 'money',
+        headerData$: this.costTotal$.pipe(map(gs => last(gs)?.value)),
+        data$: this.costTotal$,
         customOptions: this.costTotalCustomOptions,
         pipe: [true, true],
       },
       {
         label: 'Imunní',
-        icon: 'hospital' as SvgIconName,
+        icon: 'hospital',
         headerData$: this.immunized$,
-        multiLineData$: this.immunizedChart$ as Observable<ChartValue[]>,
-        customOptions: null,
+        multiLineData$: this.immunizedChart$,
+        customOptions: this.immunizedCustomOptions,
         pipe: [false, false],
       },
     ];
   }
 
   ngAfterViewInit() {
-    this.gameService.reset$.pipe(untilDestroyed(this)).subscribe(() => this.currentMitigation = undefined);
+    this.gameService.reset$.pipe(
+      untilDestroyed(this),
+    ).subscribe(() => {
+      this.currentMitigation = undefined;
+      this.mitigationNodes = [];
+    });
 
+    // TODO connect to change-only mitigation structure
+    // and make mitigationNodes a dictionary of dates
     for (const mitigation of Object.keys(MitigationsService.mitigationsI18n)) {
       this.mitigationsService.formGroup.get(mitigation)?.valueChanges.pipe(
         untilDestroyed(this),
       ).subscribe((value: any) => {
-        // TODO add multiple mitigations at a same time
-        this.currentMitigation = this.mitigationsService.getLabel(
+        const label = this.mitigationsService.getLabel(
           mitigation as keyof typeof MitigationsService.mitigationsI18n, value);
+        if (this.currentMitigation) this.currentMitigation += '\n' + label;
+        else this.currentMitigation = label;
```

Comment:
Consider to use
`this.currentMitigation = [this.currentMitigation, label].filter(Boolean).join('\n');`